packages/gasket-plugin-service-worker/package.json

```diff
@@ -33,13 +34,16 @@
   },
   "homepage": "https://github.com/godaddy/gasket/tree/master/packages/gasket-plugin-service-worker#readme",
   "dependencies": {
+    "@types/lru-cache": "^5.1.1",
+    "@types/uglify-js": "^3.13.1",
     "deepmerge": "^4.2.2",
     "lru-cache": "^5.1.1",
     "mkdirp": "^0.5.1",
     "uglify-js": "^3.5.3",
```

Comment:
Not really related to this PR, but @kinetifex -- should this move to `terser`?

Reply:
It should be updated, but we'll probably go with [swc modification](https://swc.rs/docs/configuration/minification).